src/MySqlConnector/MySqlClient/Caches/NormalizedName.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Text.RegularExpressions;
+
+namespace MySql.Data.MySqlClient.Caches
+{
+	internal class NormalizedName
+	{
+		private static readonly Regex NameRe = new Regex(@"^\s*(?:`([^`]+)`|([^\.]+)?)\s*\.?\s*(?:`([^`]+)`|([^\.]+)?)?\s*$",
```

Comment:
Wow. I feel sorry for anyone working with a database that contains back ticks in the schema name